spec/pane-axis-element-spec.coffee

```diff
@@ -0,0 +1,58 @@
+PaneAxisElement = require '../src/pane-axis-element'
+
+describe "PaneResizeHandleElement", ->
+  describe "add and remove", ->
+    class TestItemElement extends HTMLElement
+      createdCallback: ->
+        @classList.add('test-root')
+
+    TestItemElement = document.registerElement 'atom-test-item-element', prototype: TestItemElement.prototype
+
+    [paneAxis, model, Model] = []
+    beforeEach ->
+      # This is a dummy item to prevent panes from being empty on deserialization
+      class Model
```

Comment:
Rather than creating a custom `PaneAxis` model just for this test, I'd prefer to just use the real model in the specs. If you want to keep the specs focused on `PaneAxisElement`, you should be able to construct a `PaneAxis` model directly.